piston/blockchain.py

```diff
@@ -258,14 +258,14 @@ def get_block_from_time(self, timestring, error_margin=10):
         timestring_timestamp = parse_time(timestring).timestamp()
         delta = known_block_timestamp - timestring_timestamp
         block_delta = delta / 3
-        guess_block = known_block - block_delta
+        guess_block = known_block.block - block_delta
```

Comment:
Works either way. `known_block["block"]` is probably the more pythonic way of the two.